examples/sql/connect_to_motherduck.py

```diff
@@ -3,8 +3,9 @@
 # dependencies = [
 #     "altair==5.4.1",
 #     "duckdb==1.1.0",
+#     "polars==1.18.0",
+#     "pyarrow==18.1.0",
```

Comment:
hey @akshayka , would it be better to use "polars[pyarrow]>=1.18.0"
https://duckdb.org/docs/guides/python/polars.html

Reply:
Thanks! Will keep this in mind for the future. I think serializing both accomplishes the same thing?

Reply:
yes it does. Maybe just a small thing for future maintenance since we don't need to update the pyarrow version. But yeah it's fine.